🐛 BUG: "PostCSS configuration was not passed or is invalid" with svelte renderer

### What version of `astro` are you using?

0.22.1

### What package manager are you using?

yarn

### What operating system are you using?

Linux

### Describe the Bug

When rendering a page with a svelte component, the following error will show (even if I am not using postcss) : 
```
[svelte-preprocess] PostCSS configuration was not passed or is invalid. If you expect to load it from a file make sure to install "postcss-load-config" and try again.

Error: Cannot find module 'postcss-load-config'
Require stack:
- /sandbox/node_modules/svelte-preprocess/dist/transformers/postcss.js
- /sandbox/node_modules/svelte-preprocess/dist/autoProcess.js
- /sandbox/node_modules/svelte-preprocess/dist/index.js
```

This will not fail the build and the component seems to function as normal, but still floods the console.

### Link to Minimal Reproducible Example

Astro.new svelte starter example (https://codesandbox.io/s/github/snowpackjs/astro/tree/latest/examples/framework-svelte)

--- a/.changeset/large-snakes-lay.md
+++ b/.changeset/large-snakes-lay.md
@@ -1,0 +1,5 @@
+---
+'@astrojs/renderer-svelte': patch
+---
+
+Remove "PostCSS configuration was not passed or is invalid" error when using the Svelte renderer

--- a/packages/renderers/renderer-svelte/index.js
+++ b/packages/renderers/renderer-svelte/index.js
@@ -20,7 +20,6 @@ export default {
 							less: true,
 							sass: { renderSync: true },
 							scss: { renderSync: true },
-							postcss: true,
 							stylus: true,
 							typescript: true,
 						}),

